ADOPTERS.md

```diff
@@ -0,0 +1,21 @@
+# Paketo Buildpacks Adopters
+
+If you're using Paketo Buildpacks and want to add your organization to this list, just add a response to this Discussion and we'll get your organization added.
+
+If you want to add it yourself, please follow these directions
+
+## Success Stories
+
+Below is a list of adopters of Paketo Buildpacks in **production environments** that have
+publicly shared the details of how they use it.
```

Comment:
What is allowed to be included here? Is it just a text list of the company names? Are companies allowed to include a link, perhaps to a blog post or announcement about how they are using it? 

If we are allowing a link, I think we should spell out that the link must point to either the company's home page or an announcement directly mentioning their usage of Paketo buildpacks. Just to prevent folks from trying to link to sales material or other things.